src/hooks/useI18n.js

```diff
@@ -9,6 +9,7 @@ export const useI18n = () => {
 
   return {
     _: window._,
+    _n: window._n,
```

Comment:
As discussed in #1179, it may be required to defined `_n` as a reserved name. Otherwise it could be renamed in the minimized bundle, and the translation keys would not be parsed.  

https://github.com/Qwant/erdapfel/blob/1e4919d4592df3d50d60b339c62364473ccfe0fc/build/webpack.config.js#L97-L99

Reply:
ok, see https://github.com/Qwant/erdapfel/pull/1180